config.yaml

```diff
@@ -31,15 +31,15 @@
           ensemble_gene_id: ensembl_gene_id
           hgnc_gene_id: hgnc_symbol
         destination: *dest
-        
+
     - proteomics_tmt:
         files:
           - name: agora_proteomics_tmt
-            id: syn32188234
+            id: syn35221005
             format: csv
         final_format: json
         provenance:
-          - syn32188234
+          - syn35221005
         column_rename:
```

Comment:
The new source file syn35221005 does not need most of these column_renames. 

Example:
syn35221005 doesn't have a peptide column, but it does have a uniqid column (remember that all colnames are automatically lowercased by the framework), so we don't need the `peptide: uniqid` column_rename.

Please reconcile the column names in syn35221005 with the list of column_rename and remove the ones we don't need. The same changes should be applied to test_config.yaml.